infra/README.md

```diff
@@ -54,7 +54,7 @@ For now, the following steps are necessary to configure the cluster:
 - 1. ssh to the manager machine
 - 2. `cd ~/testground-aws-setup/infra/`
 - 3. `git pull` (get latest scripts)
-- 2. `cd ansible`
+- 2. `cd aws/ansible`
```

Comment:
It's actually just `ansible`. The `aws/ansible` is just leftovers from a previous version of the branch that is now by default shipped because the packer images hasn't been updated.

Reply:
```suggestion
- 2. `cd ansible`
```